channels_irc/server.py

```diff
@@ -0,0 +1,69 @@
+import asyncio
+import traceback
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class BaseServer:
+    """
+    Base class for common interface server functions, including
+    creating an application, sending to the consumer, and application_checking/
+    error handling
+    """
+    def _send_application_msg(self, msg):
+        """
+        sends a msg (serializable dict) to the appropriate Django channel
+        """
+        return self.application_queue.put_nowait(msg)
+
+    def noop_from_consumer(self, msg):
+        """
+        empty default for receiving from consumer
+        """
```

Comment:
needs a `pass`?

Reply:
technically, no. I relatively recently learned (from some other libraries) that a comment string is as good as a pass, as far as noop goes...